options.go

```diff
@@ -1180,6 +1180,37 @@ func (opts *Options) SetAtomicFlush(value bool) {
 	C.rocksdb_options_set_atomic_flush(opts.c, C.uchar(btoi(value)))
 }
 
+// SetMaxSubcompactions represents the maximum number of threads that will
+// concurrently perform a compaction job by breaking it into multiple,
+// smaller ones that are run simultaneously.
+//
+// Default: 1 (i.e. no subcompactions)
+func (opts *Options) SetMaxSubcompactions(value uint32) {
+	C.rocksdb_options_set_max_subcompactions(opts.c, C.uint32_t(value))
+}
+
+// GetMaxSubcompactions gets the maximum number of threads that will
+// concurrently perform a compaction job by breaking it into multiple,
+// smaller ones that are run simultaneously.
+func (opts *Options) GetMaxSubcompactions() uint32 {
+	return uint32(C.rocksdb_options_get_max_subcompactions(opts.c))
+}
+
+// SetMaxBackgroundJobs maximum number of concurrent background jobs
+// (compactions and flushes).
+//
+// Default: 2
+//
+// Dynamically changeable through SetDBOptions() API.
+func (opts *Options) SetMaxBackgroundJobs(value int) {
```

Comment:
NB: Now that we have these, can we mark `SetMaxBackgroundCompactions` and `SetMaxBackgroundFlushes` as deprecated? According to [this](https://go.dev/blog/godoc) page:

> Sometimes a struct field, function, type, or even a whole package becomes redundant or unnecessary, but must be kept for compatibility with existing programs. To signal that an identifier should not be used, add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.

I think this will make Goland/VSCode also mark it as such in autocomplete.